best-practices.md

```diff
@@ -13,20 +11,14 @@
 
 ## Proper use of EventEmitters
 
-EventEmitter extends from Subject, which give us the ability to subscribe. Should we? It's easy, convenient and super cool!
-But we should not! EventEmitter is an angular2 abstraction to communicate between components. Use it only with `@Output()`.
-It's not sure that it will continue being an Observable nor that will continue being part of the API at all! 
-
+While EventEmitter extends from Subject, giving us the ability to subscribe, you should not count on this staying. [Ward Bell has stated](http://www.bennadel.com/blog/3038-eventemitter-is-an-rxjs-observable-stream-in-angular-2-beta-6.htm#comments_47949) that it is highly likely to be removed before release. EventEmitter is intended to ONLY be used with `@Output()`. If you want to subscribe just use a Subject directly.
 
 See more 
 - [What is the proper use of an EventEmitter?](http://stackoverflow.com/questions/36076700/what-is-the-proper-use-of-an-eventemitter)
 
-
-
-
 ## ElementRef, nativeElement and Renderer
 
-Official documentation states we should not access nativeElement directly, how should we do it then? To access `nativeElement` correctly we must use Renderer since it's WebWorker safe.
+Official documentation recommends against using `nativeElement` directly. The reason for this is because it is not WebWorker safe (It may also not work with Universal, and alternate renderer implementaitons). The preferred approach is to use the `nativeElement` with the Renderer. 
 

```

Comment:
maybe add something like "unless you are only targeting desktop browsers" just to make it clear that it is often OK (sometimes, it's the only approach possible)